.buildkite/pipeline.yml

```diff
@@ -592,7 +592,10 @@ steps:
         artifact_paths: "diagnostic_edmfx_aquaplanet/*"
         agents:
           slurm_mem: 20GB
-      
+
```

Comment:
This was just a minor fix of the pipeline where one prognostic edmf case was put in the wrong group before.